src/middleware/routeHandler.ts

```diff
@@ -0,0 +1,250 @@
+import { backOff } from "exponential-backoff"
+import simpleGit from "simple-git"
```

Comment:
> we were not using the rollback properly,

What does "properly" mean?

Reply:
simply put, we were not giving the correct types that `backoff` expected